src/main/java/gov/cabinetoffice/gap/applybackend/config/SpotlightQueueConfigProperties.java

```diff
@@ -0,0 +1,21 @@
+package gov.cabinetoffice.gap.applybackend.config;
+
+import lombok.*;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+import javax.validation.constraints.NotNull;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@Configuration
+@ConfigurationProperties(prefix = "spotlight-queue")
+public class SpotlightQueueConfigProperties {
+
+    @NotNull
```

Comment:
Not sure this even works. Didn't throw an error when I spelled the property name wrong...